modules/developers-guide/examples/bash/node-npm.sh

```diff
@@ -0,0 +1 @@
+npm i @stargate-oss/stargate-grpc-node-client
```

Comment:
I see [the staging page](https://coppi.sjc.dsinternal.org/en/grpc/stargate/1.0/developers-guide/gRPC-node-client.html) linked in the PR description has the old package location - `@stargate` - but here it's right - `@stargate-oss`. Are the staging docs out of date?

Reply:
fixed